src/content/admin/resource-manager.mdx

```diff
@@ -32,37 +32,21 @@ It calculates the average usage and automatically adjusts the resource requests
 1. Recommendation Generation: The Resource Manager generates new resource requests based on the average usage, applying a correction factor to ensure efficiency
 1. Automatic Application: When a new pod is created, the recommended resource requests are automatically applied unless overridden the annotation `dev.okteto.com/resource-manager: false`
 
-## Enabling and Configuring the Resource Manager
-<!-- ### Enabling the Resource Manager
-The Resource Manager is controlled through the Helm chart configuration. By default, it is enabled with the following settings:
-
-``` yaml
-resourceManager:
-  enabled: true
-  schedule: 300
-  deletePeriod: 15
-  priorityClassName:
-    labels:
-	annotations:
-	resources:
-	recommendations:
-		weight:
-		correction:
-		min:
-			cpu:
-			memory:
-``` -->
+## Configuring the Resource Manager
+### Enabling the Resource Manager
+Resource Manager has been installed in Okteto chart version 1.26 and above. Refer to our [Helm configuration documentation](self-hosted/helm-configuration.mdx#resourcemanager) for available options.
```

Comment:
sharing in case we missed it: RM is also included in 1.25 (not installed by default and without admin view)

Reply:
also, I would say something like "Resource Manager is available from Okteto chart version 1.25 and above.". I think it's confusing to use both "installed" and "enabled". 

Installed and enabled are the same thing in the context of Resource Manager. Once you set `resourceManager.enabled: true` it will create the cronjob in your cluster. That is what we consider the "installation" or "enabling" resource manager.

Reply:
+1